time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -256,9 +256,28 @@ def post(self):
         return time_entries_dao.create(ns.payload), HTTPStatus.CREATED
 
 
+# TODO: Once this endpoint is working as expected in prod, review and
+# remove unnecessary filter args. As we are using same attributes_filter
```

Comment:
@PaulRC-ioet  Let's remove these lines and use a new attributes_filter for this endpoint.

Reply:
the attribute filters were removed and the TODO line deleted